_4angle.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class _4angle : _BaseClass
+    {
+        private protected int x1, y1;
+        private protected int x2, y2;
+        private protected int x3, y3;
+        private protected int x4, y4;
+
+        public int x_x1 { get { return x1; } set { x1 = value; } }
```

Comment:
you can simplify this by 
```suggestion
        public int X1 { get; set; }
```
That's the same after compilation